metadata/src/lib.rs

```diff
@@ -214,7 +214,10 @@ impl Metadata {
         MetadataHasher::new(self)
     }
 
-    /// Filter out any pallets that we don't want to keep, retaining only those that we do.
+    /// Filter out any pallets and/or runtime_apis that we don't want to keep, retaining only those that we do.
+    /// Note:
```

Comment:
I would rather create a dev-note using `//` to avoid this to end-up in the API docs